packages/components/src/Drawer/Drawer.component.js

```diff
@@ -213,7 +213,9 @@ function DrawerContent({ children, className, ...rest }) {
 			className={classnames('tc-drawer-content', theme['tc-drawer-content'], className)}
 			{...rest}
 		>
-			{children}
+			<div className={classnames('tc-drawer-content-wrapper', theme['tc-drawer-content-wrapper'])}>
```

Comment:
Why did you add this div level ? The associated css is 
```
width: $tc-drawer-content-max-width;
margin: 0 auto;
```
Before your change, this was applied to the `tc-drawer-content` just above